datastore/longhorn.go

```diff
@@ -607,7 +604,12 @@ func (s *DataStore) GetNode(name string) (*longhorn.Node, error) {
 		}
 		return nil, err
 	}
-	return result.DeepCopy(), nil
+	node := result.DeepCopy()
+	if node.Status.Conditions == nil {
+		node.Status.Conditions = map[types.NodeConditionType]types.Condition{}
+		node.Status.Conditions[types.NodeConditionTypeMountPropagation] = types.GetNodeConditionFromStatus(node.Status, types.NodeConditionTypeMountPropagation)
```

Comment:
I see now.

But why deal with MountPropagation in special case here? There are other conditions for the node as well.

Caller should able to deal with the empty map.

Reply:
I think driver would call before node controller update conditions, so I add the unknown status for mountPropagation for it.

Reply:
Yeah, then the driver can treat `not found` as `unknown` as well.

Reply:
ok. I will fix this.